docs/source/README.md

```diff
@@ -39,6 +41,14 @@ spec:
 EOF
 ```
 
+### Install without the RabbitMQ Operators
+
+* You will need a RabbitMQ instance running and accessible via an URL/IP
+
+* Install everything except the [RabbitMQ Cluster Operator](https://github.com/rabbitmq/cluster-operator)
+
+* Note: An external RabbitMQ instance can be used, but if you want to use the `Source` without predeclared resources (specifically the `Exchange` and `Queue`), the `RabbitMQ Message Topology Operator` needs to be installed in the external cluster too.
```

Comment:
The message topology operator isn't installed on the "external cluster". It's installed on the Kubernetes cluster.